### ISSUE TITLE
Set Suffix to ".pug", inherited pug files automatically look for ".jade" files

### ISSUE THREAD

**Issue by KarmaQueen** [neuland/jade4j#166]
I have two pug files, index.pug and ./layout/layout.pug

index.pug extends layout.pug.
```
extends ./layout/layout.pug
block ...
```

Jade4J will correctly find index.pug, but will crash, saying it cannot find "index.pug.jade".

```
de.neuland.jade4j.exceptions.JadeParserException: the template [./layout/layout.pug.jade] could not be opened
class path resource [templates/layout/layout.pug.jade] cannot be opened because it does not exist in index.pug:1
	at de.neuland.jade4j.parser.Parser.createParser(Parser.java:436) ~[jade4j-1.2.5.jar:na]
	at de.neuland.jade4j.parser.Parser.parseExtends(Parser.java:419) ~[jade4j-1.2.5.jar:na]
	at de.neuland.jade4j.parser.Parser.parseExpr(Parser.java:109) ~[jade4j-1.2.5.jar:na]
	at de.neuland.jade4j.parser.Parser.parse(Parser.java:69) ~[jade4j-1.2.5.jar:na]
	at de.neuland.jade4j.JadeConfiguration.createTemplate(JadeConfiguration.java:87) ~[jade4j-1.2.5.jar:na]
	at de.neuland.jade4j.JadeConfiguration.getTemplate(JadeConfiguration.java:65) ~[jade4j-1.2.5.jar:na]
	at de.neuland.jade4j.spring.view.JadeView.getTemplate(JadeView.java:67) [spring-jade4j-1.2.5.jar:na]
	at de.neuland.jade4j.spring.view.JadeView.doRender(JadeView.java:59) [spring-jade4j-1.2.5.jar:na]
	at de.neuland.jade4j.spring.view.JadeView.renderMergedTemplateModel(JadeView.java:30) [spring-jade4j-1.2.5.jar:na]
	at org.springframework.web.servlet.view.AbstractTemplateView.renderMergedOutputModel(AbstractTemplateView.java:167) [spring-webmvc-4.3.11.RELEASE.jar:4.3.11.RELEASE]
	at org.springframework.web.servlet.view.AbstractView.render(AbstractView.java:303) [spring-webmvc-4.3.11.RELEASE.jar:4.3.11.RELEASE]
	at org.springframework.web.servlet.DispatcherServlet.render(DispatcherServlet.java:1286) [spring-webmvc-4.3.11.RELEASE.jar:4.3.11.RELEASE]
	at ...
```

Of course, I could just rename all my files to *.jade to fix the issue, but I hope to be able to use *.pug files in the future with jade4j.

Here is my Jade configuration file:
```
@Configuration
@EnableWebMvc
public class JadeConfig {

	@Bean
	public SpringTemplateLoader templateLoader() {
		SpringTemplateLoader templateLoader
				= new SpringTemplateLoader();
		templateLoader.setBasePath("classpath:/templates/");
		templateLoader.setSuffix(".pug");
		return templateLoader;
	}

	@Bean
	public JadeConfiguration jadeConfiguration() {
		JadeConfiguration configuration
				= new JadeConfiguration();
		configuration.setCaching(false);
		configuration.setPrettyPrint(true);
		configuration.setTemplateLoader(templateLoader());
		return configuration;
	}

	@Bean
	public ViewResolver viewResolver() {
		JadeViewResolver viewResolver = new JadeViewResolver();
		viewResolver.setConfiguration(jadeConfiguration());
		viewResolver.setSuffix(".pug");
		return viewResolver;
	}
}
```

**Comment by chbloemer** [comment]
You are using Jade4j 1.2.5. Please try out Jade4j 1.2.7 and Spring-jade4j 1.2.7. This should be fixed in the newer versions.

**Comment by KarmaQueen** [comment]
This indeed has been solved in 1.2.7.
